project/Dependencies.scala

```diff
@@ -86,16 +87,16 @@ object Dependencies {
   val slf4jIt             = "org.slf4j"          % "slf4j-simple"               % V.slf4j
 
   // java
-  val slf4jSimple   = "org.slf4j" % "slf4j-simple"   % V.slf4j
-  val slf4jApi      = "org.slf4j" % "slf4j-api"      % V.slf4j
-  val azureIdentity = "com.azure" % "azure-identity" % V.azureSdk
-
-  val badrows      = "com.snowplowanalytics" %% "snowplow-badrows"                      % V.badrows
-  val tracker      = "com.snowplowanalytics" %% "snowplow-scala-tracker-core"           % V.tracker
-  val trackerEmit  = "com.snowplowanalytics" %% "snowplow-scala-tracker-emitter-http4s" % V.tracker
-  val schemaDdl    = "com.snowplowanalytics" %% "schema-ddl"                            % V.schemaDdl
-  val igluClient   = "com.snowplowanalytics" %% "iglu-scala-client"                     % V.igluClient
-  val analyticsSdk = "com.snowplowanalytics" %% "snowplow-scala-analytics-sdk"          % V.analyticsSdk
+  val slf4jSimple   = "org.slf4j"              % "slf4j-simple"                          % V.slf4j
+  val slf4jApi      = "org.slf4j"              % "slf4j-api"                             % V.slf4j
+  val azureIdentity = "com.azure"              % "azure-identity"                        % V.azureSdk
+  val protobuf      = "com.google.protobuf"    % "protobuf-java"                         % V.protobuf
```

Comment:
Please can you annotate that this is a dependency override?  So we don't accidentally remove it by thinking it is not necessary.  And so we don't forget to remove it when it really becomes not necessary?

Reply:
Done !